partcad-cli/src/partcad_cli/click/commands/test.py

```
@@ -1,3 +1,24 @@
+#
+
+
+import rich_click as click
+import partcad.utils as pc_utils
+
+
+@click.command()
+@click.pass_obj
+def cli(ctx):
+    with pc.logging.Process("Test", "this"):
+        cli_test(args, ctx)
```

Ruff (0.8.2)

Undefined name `args`

(F821)